test/sanity/globalConstants.wtest

```diff
@@ -0,0 +1,103 @@
+const x = 1 + 1
+test "global constants are referenceable" {
+  assert.equals(2, x)
+}
+
+
+const a = b
+const b = 1 + 1
+const c = b
```

Comment:
My guess is this could be achieved by somehow sorting the initializations based on the direct references.
So, in this example the order to use might be: `b`, `a`, `c`

Reply:
I don't think this example is interesting. Maybe it will work anyway if you solve this:
```
const a = new C(x=b)
const b = new C(x=a)
```
But the hoysting of b to be referenced by a is not necessary, easily avoided by programmer, wouldn't care about this exact case, unless we can find something more specific.